lib/Parser/LineDataParser.php

```diff
@@ -217,12 +210,67 @@ public function parseDefinitionList(array $lines): DefinitionList
         return new DefinitionList($definitionList);
     }
 
+    /**
+     * @param string[] $lines
+     *
+     * @return FieldOption[]
+     */
+    public function parseFieldList(array $lines): array
+    {
+        $listItems          = [];
+        $currentFieldOption = null;
+        foreach ($lines as $line) {
+            $newFieldOption = $this->parseFieldOption($line);
+            if ($newFieldOption === null) {
+                if ($currentFieldOption === null) {
+                    throw new RuntimeException('Expected to find field option at start of field list.');
+                }
+
+                $currentFieldOption->appendLine($line);
+            } else {
+                if ($currentFieldOption !== null) {
+                    $currentFieldOption->setNodes($this->getSubNodes($currentFieldOption->getBody()));
+                    $listItems[] = $currentFieldOption;
+                }
+
+                $currentFieldOption = $newFieldOption;
+            }
+        }
+
+        if ($currentFieldOption !== null) {
+            $currentFieldOption->setNodes($this->getSubNodes($currentFieldOption->getBody()));
+            $listItems[] = $currentFieldOption;
+        }
+
+        return $listItems;
+    }
+
     public function parseFieldOption(string $line): ?FieldOption
     {
         if (preg_match('/^(?P<offset>\s*):(?P<name>(?:\\\\:\s|[^:]|:\S)+):(?: +(?P<body>.+)|)$/', $line, $match) > 0) {
-            return new FieldOption($match['name'], strlen($match['offset']), $match['body'] ?? '');
+            $fieldOption = new FieldOption($match['name'], strlen($match['offset']), $match['body'] ?? '');
+            $fieldOption->setNodes($this->getSubNodes($fieldOption->getBody()));
+
+            return $fieldOption;
         }
 
         return null;
     }
+
+    /** @return array|Node[] */
```

Comment:
`Node[]` is an `array`, so this is as good as `array` :thinking:

Reply:
Oh yeah, my IDE is sometimes doing these strange automatic comments...